routes/api.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Route;
+
+/*
+|--------------------------------------------------------------------------
+| API Routes
+|--------------------------------------------------------------------------
+|
+| Here is where you can register API routes for your application. These
+| routes are loaded by the RouteServiceProvider within a group which
+| is assigned the "api" middleware group. Enjoy building your API!
+|
+*/
+
+Route::middleware('auth:api')->get('/user', function (Request $request) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Opening parenthesis of a multi-line function call must be the last content on the line](https://app.codacy.com/gh/BuildForSDG/Team-059-Product/pullRequest?prid=5478046)